core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/content/services/html/libraries/iframe-edit-mode.css.ts

```diff
@@ -454,6 +454,10 @@ export const getEditPageCss = (timestampId: string): string => {
         display: block;
     }
     
+    ${timestampId} [data-inode][data-field-name][data-content].dotcms__inline-edit-field {
```

Comment:
This mean we are forcing the user to add a class to the `div`, we have multiples data-attr we can use for this.

Reply:
following the same approach as inline editing.